README.md

```diff
@@ -4,27 +4,74 @@
 
 https://jotai.org/docs/integrations/scope
 
-## Pro Tips
+## Scoped Atom resolution rules
+1. Each ScopeProvider creates a new scope pool.
+2. If a primitive atom is scoped, its scoped copy will be stored within the scope pool.
+3. If a derived atom is scoped, itself and all of its dependencies will be stored within the scope pool.
```

Comment:
I think it's straightforward and mostly controllable.
I didn't follow the detailed discussion, but I wonder if you have considered this situation.
```js
const a = atom(1);
const b = atom((get) => get(a) * 2);
const c = atom((get) => get(a) * 3);
```
if `b` is explicitly scoped, `a` is implicitly scoped too, and thus `c` is also practically scoped, right?

Reply:
- If b is scoped
a -> global
b -> scoped
c -> global

- if a and b are scoped
a -> scoped
b -> scoped
c -> scoped (implicitly)

Reply:
Okay, so unless `a` is explicitly scoped, `c` is global. Nice.

Reply:
Fantastic!

Reply:
If `b` is scoped, can I write to `b` and have it update implicit scoped `a`?